frontend/pages/discover/projects.tsx

```diff
@@ -32,7 +32,7 @@ import { Project as ProjectType } from 'types/project';
 
 import { getProject } from 'services/projects/projectService';
 
-export const getServerSideProps = withLocalizedRequests(async ({ locale }) => {
+export const getStaticProps = withLocalizedRequests(async ({ locale }) => {
```

Comment:
If you apply a filter and reload the page, you will see that we will first render the page with all the projects (the request to fetch the projects is sent without the filter), and then it is rendered with the filter applied (another request with the filter is sent).

This is because `useQueryParams` uses `useRouter` internally and the [Next.js' documentation](https://nextjs.org/docs/api-reference/next/router#router-object) says this:
> `query`: `Object` - The query string parsed to an object, including [dynamic route](https://nextjs.org/docs/routing/dynamic-routes) parameters. It will be an empty object during prerendering if the page doesn't use [Server-side Rendering](https://nextjs.org/docs/basic-features/data-fetching/get-server-side-props). Defaults to `{}`

To avoid `query` being an empty object, we want to keep `getServerSideProps` in this file.